Legion.FrontEnd/src/events/EventsCalendar/Components/Month.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Week from './Week';
+import moment from 'moment';
+
+class Month extends React.PureComponent {
+  state = {
+    month: this.props.month,
+    selected: this.props.selected
+  }
+
+  createWeeks = (month) => {
+    var weeks = [],
+      date = month.clone().startOf('month').startOf('week'),
+      // startDate = date.clone(),
```

Comment:
Remove commented out code unless there a good reason to keep it, in which case, add a comment above explaining why you're keeping it.

Reply:
Will do. I think i was testing something, and forgot about it. Thanks for catching it.